lib/spack/env/cray/cc

```diff
@@ -1 +0,0 @@
-../cc
```

Comment:
what you can do to remove those is:
```
git reset --soft develop
```
and then un-stage changes for these two files.

Reply:
btw, I am on mac high sierra and I don't have those issues...

Reply:
Oh, I think I just figured out. You have a case-sensitive file system, right? I'm using a standard APFS, which is case-preserving but case-insensitive, so CC and cc cannot coexist in the same folder.

Reply:
i think you are right, i can't check now as `diskutil info /` is not particularly descriptive anymore:
```
   File System Personality:  APFS
   Type (Bundle):            apfs
   Name (User Visible):      APFS
```
(in Sierra one might see `Case-sensitive Journaled HFS`)

Reply:
If you use the normal disk utility GUI, click on the partition and then in info, a list will pop up like:

```
Volume name : Macintosh HD
Volume type : APFS Volume
BSD device node : disk1s1
Mount point : /
File system : APFS (Encrypted)
Connection : PCI-Express
Device tree path : IODeviceTree:/PCI0@0/RP01@1C/SSD0@0/AppleS3XController
Writable : Yes
**Is case-sensitive : No**
File system UUID : 74733FAD-312D-3A96-99C7-67378A53706B
etc...
```

Reply:
thanks, i checked and i have the same: `Is case-sensitive: No`.

Reply:
And, just to clarify, in your system CC and cc coexist in the lib/spack/env/cray/cc folder despite the FS being case-insensitive?

Because I did a quick test and in my system, this happens:
```
$ touch CC
$ echo "test" > cc
$ cat CC
test
```
and I do a `touch cc`, the time of CC is updated. My system is completely blind to upper and lower case.

Reply:
> in your system CC and cc coexist in the lib/spack/env/cray/cc folder despite the FS being case-insensitive?

nope:
 
```
$ ls ~/spack/lib/spack/env/cray
cc	ftn
```